region/maridia/outer.json

```diff
@@ -550,215 +597,35 @@
-                  ]},
-                  {"and": [
-                    "canSuitlessMaridia",
-                    "canCrabClimb",
-                    "Super"
-                  ]}
-                ]}
-              ],
-              "unlock": null
-            },
-            {
-              "id": 5,
-              "requires": [ "never" ],
-              "unlock": null
-            },
-            {
-              "id": 6,
-              "requires": [
-                {"or": [
-                  {"canShineCharge": {
-                    "usedTiles": 24
                   }},
                   {"canComeInCharged": {
-                    "fromNode": 3,
-                    "framesRemaining": 150
+                    "fromNode": 2,
+                    "framesRemaining": 120
                   }}
-                ]},
-                "canSwim"
-              ],
-              "unlock": null
-            },
-            {
-              "id": 7,
-              "requires": [ "never" ],
-              "unlock": null
-            }
-          ]
-        },
-        {
-          "from": 4,
-          "to": [
-            {
-              "id": 1,
-              "requires": null,
-              "unlock": null
-            },
-            {
-              "id": 2,
-              "requires": null,
-              "unlock": null
-            },
-            {
-              "id": 3,
-              "requires": null,
-              "unlock": null
-            },
-            {
-              "id": 5,
-              "requires": [ "never" ],
-              "unlock": null
-            },
-            {
-              "id": 6,
-              "requires": [
-                "canSwim",
-                {"canShineCharge": {
-                  "usedTiles": 24
-                }}
+                ]}
               ],
-              "unlock": null
-            },
-            {
-              "id": 7,
-              "requires": [ "never" ],
-              "unlock": null
-            }
-          ]
-        },
-        {
-          "from": 5,
-          "to": [
-            {
-              "id": 1,
-              "requires": [ "never" ],
-              "unlock": null
-            },
-            {
-              "id": 2,
-              "requires": [ "never" ],
-              "unlock": null
-            },
-            {
-              "id": 3,
-              "requires": [ "never" ],
-              "unlock": null
-            },
-            {
-              "id": 4,
-              "requires": [ "never" ],
-              "unlock": null
-            },
-            {
-              "id": 6,
-              "requires": [ "never" ],
-              "unlock": null
-            },
-            {
-              "id": 7,
-              "requires": [ "Morph" ],
-              "unlock": null
+              "unlock": null,
+              "note": "FIXME There are ways to do this suitless if you can come in charged, especially if you can carry momentum, but how to test this?"
```

Comment:
I would leave out the one from node 2(right door) completely but the bot one looks good with framesRemaining 120 with no momentum to speak of.

Reply:
I ran some tests by taking out gravity suit after establishing a full-speed charge in the adjacent room, and it's possible to store the spark and use it. I think we could have door 2 by requiring 180 frames (so basically need to establish the entire charge in the adjacent room). I would be inclined to do that, unless you have objections.

Reply:
I tried the 120 suitless and it seems doable. That means 120 with gravity is way too lenient. After more tests on that, I'll bring it down to 80.

Reply:
I removed the water property in the adjacent room with smile and while I couldn't do it sitting at the very last second before door transition, I  could get it done sitting before the door and using the momentum of the two door tiles. I would say you're good to go.